src/services/utilities.ts

```diff
@@ -1358,4 +1358,115 @@ namespace ts {
             diagnostics: error ? concatenate(diagnostics, [error]) : diagnostics
         };
     }
-}
+
+    export function getMissingAbstractMembersInsertion(classDecl: ClassDeclaration, resolvedType: ResolvedType, checker: TypeChecker, newlineChar: string): string {
+        const classSymbol = checker.getSymbolOfNode(classDecl);

```

Comment:
It shouldn't return undefined. I've checked the case of exporting a default class (there is a symbol named `default` in that case) and handling class expressions (they are anonymous, but still have a symbol).

However, I found an issue with the latter case where we don't get the implements type nodes correctly. Looking into this further.